core/metacling/src/TCling.cxx

```diff
@@ -626,6 +630,18 @@ extern "C" int TCling__AutoLoadCallback(const char* className)
    return ((TCling*)gCling)->AutoLoad(className);
 }
 
+extern "C" int TCling__AutoLoadLibrary(const char* Name)
```

Comment:
Could you inject `Module` in the function name, to clarify that it's only used in the runtime modules case? E.g. `TCling__AutoLoadLibraryForModules`.